lib/archethic_web/router.ex

```diff
@@ -71,6 +71,8 @@ defmodule ArchEthicWeb.Router do
       :last_transaction_content
     )
 
+    get("/origin_key/:origin_public_key", ArchEthicWeb.API.TransactionController, :origin_key)
```

Comment:
I think a post query would make more sense here

Reply:
Look at the fee API.
The goal is to protect a bit the information transmitted do not be visible from the logs, caches, histories.
GET requests aims to be shareable, so any person can reproduce it and see the result and here it's not the case.